package.json

```diff
@@ -16,25 +16,23 @@
 		"type": "git",
 		"url": "https://github.com/savoirfairelinux/vscode-bitbake.git"
 	},
-	"workspaces": [
-		"client",
-		"server"
-	],
 	"scripts": {
+		"postinstall": "cd server && npm install --omit=dev && cd ../client && npm install --omit=dev",
 		"fetch": "npm run fetch:poky",
 		"fetch:poky": "mkdir -p resources/poky && curl -L -o resources/poky.tar.bz2 https://downloads.yoctoproject.org/releases/yocto/yocto-4.2.3/poky-aa63b25cbe25d89ab07ca11ee72c17cab68df8de.tar.bz2 && tar -xvjf resources/poky.tar.bz2 -C resources",
-		"compile": "tsc -b",
+		"compile": "tsc -b && npm run compile:server && npm run compile:client",
```

Comment:
I had to remove the workspace property since it could not generate client/node_modules which is required by vsce.
The presented package.json architecture fulfills current needs. I suggest merging it that way and if new needs arise later, modify it.

Reply:
The current package.json fulfills the packaging and testing needs we have while following the example documented example for VSCode language server extensions. I suggest merging this version and mofifying it when new needs arise.